com.bda.controller/build.gradle

```diff
@@ -0,0 +1,2 @@
+configurations.maybeCreate("default")
```

Comment:
Bah, it's a shame this has to go at the root - we can't at least shove it inside android/?

-[Unknown]